src/server/api/routers/bot-source.ts

```diff
@@ -51,6 +52,33 @@ function deleteByIdHandler() {
     })
 }
 
+function toggleVisibility() {
```

Comment:
Let's not define it toggle. 
Let's define function setVisibility
FE would set it true/false directly instead.

Reply:
updated